Document/0x05b-Basic-Security_Testing.md

```diff
@@ -247,20 +248,22 @@ You'll come across different use cases on how you can use adb commands when test
 
 ##### Angr
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1235>
+Angr is a Python framework for analyzing binaries. It is useful for both static and dynamic symbolic ("concolic") analysis. In other words: given a binary and a requested state, Angr will try to get to that to the requested state in the binary, using formal methods to find a path, as well as brute forcing. This can make reverse-engineerig "I just want a solution style" - way easier than with for instance IDA Pro, Radare2, or Ghidra. Angr operates on the VEX intermediate language and comes with a loader for ELF/ARM binaries, so it is perfect for dealing with _native_ Android binaries.
```

Comment:
- This can make reverse-engineerig "I just want a solution style" - way easier than with for instance IDA Pro, Radare2, or Ghidra. -> two things: 1) I would rephrase the first sentence but don't know why now. 2) why is it easier?

- Angr operates on the VEX intermediate language and comes with a loader for ELF/ARM binaries, so it is perfect for dealing with _native_ Android binaries. -> do you mean Android native code? Maybe: "Android native code such as native libraries"?

Reply:
1.: Is `This can make reverse-engineerig "I just want a solution style" - way easier than with for instance IDA Pro, Radare2, or Ghidra.`  --> `Using angr to get to the requested state is way faster than taking manual steps for debugging and searching the path towards the required state.` better?
2.: yes, i meant Android native code such as native libraries. Will fix that

Reply:
Yes, I like that!